src/engines/aurora/kotorjadegui/listbox.cpp

```diff
@@ -352,4 +352,14 @@ void WidgetListBox::positionItemWidgets() {
 	}
 }
 
+void WidgetListBox::applyChangesToItemWidgets() {
+	for (int i = 0; i < _itemWidgets.size(); ++i) {
```

Comment:
`comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<Engines::WidgetProtoItem*>::size_type’`

(I.e. i should be a size_t)